WcaOnRails/app/jobs/sync_mailing_lists_job.rb

```diff
@@ -25,5 +72,18 @@ def perform
     User.clear_receive_delegate_reports_if_not_staff
     GsuiteMailingLists.sync_group("reports@worldcubeassociation.org", User.delegate_reports_receivers_emails)
     GsuiteMailingLists.sync_group("advisory@worldcubeassociation.org", Team.wac.current_members.includes(:user).map(&:user).map(&:email))
+
+    SENIOR_DELEGATES_REGIONS_INFO.each do |region|
+      senior_delegates = User.senior_delegates.where("region like ?", region[:query])
+      if senior_delegates.length > 1
+        raise "Weird, I found multiple senior delegates that match #{region[:query]}"
+      elsif senior_delegates.length < 1
+        raise "I could not find any senior delegates that matches #{region[:query]}"
```

Comment:
matches -> match

Reply:
I actually reworded the sentences a bit, I'll push the new ones now!